library/tasks/Hisat2.wdl

```diff
@@ -46,35 +46,38 @@ task HISAT2PE {
       -p 4 -S ${output_name}.sam 
     samtools sort -@ 4 -O bam -o "${output_name}.bam" "${output_name}.sam" 
   }
+
   runtime {
-    docker:"quay.io/humancellatlas/secondary-analysis-hisat2:v0.2.2-2-2.1.0"
-    memory:"5 GB"
+    docker: "quay.io/humancellatlas/secondary-analysis-hisat2:v0.2.2-2-2.1.0"
```

Comment:
There was some discussion of making dockers parameterized as inputs. Is this something we want to either (1) show in the guide or (2) standardize?